### DISCUSSION TITLE
get package.json version as env variable

### DISCUSSION THREAD

**Discussion by wiulma** [goatandsheep/react-native-dotenv#409] [General]
Hi! is it possible in some way to read the package.json version and set it as an env variable during the compile time? I'd like to avoid to import the package.json in the codebase to get the version number. I'd like to diaplay it in the info screen. 
Any suggestions? thanks a lot :-)

**Comment by goatandsheep** [comment]
```javascript
const { version } = require('../package.json')
```

**Reply by wiulma** [reply]
Hi! thanks for your reply. 
Anyway I was looking for a better option to be honest.
I woudln't like to import the package.json.
I was thinking about something during the build time, in order to add an env variable to the ones defined in the .env file.
Something like a script step during the android and ios build flow.
Any other option from your point of view? thanks :-)

**Reply by goatandsheep** [reply]
If you read the package info like above in your babel.config.js and then say

```javascript
process.env.VERSION = version
```

It should work. This has to happen in this file because this library is a Babel plugin and so it will not inject any values after Babel is run.
